SUPPORT.md

```diff
@@ -6,7 +6,7 @@
 
 This project uses GitHub Issues to track bugs and feature requests. Please search the existing issues before filing new issues to avoid duplicates. For new issues, file your bug or feature request as a new issue.
 
-Issues can be created and searched through for existing [issues here](https://github.com/Azure/terraform-azurerm-avm-ptn-virtualwan/issues).
+Issues can be created and searched through for existing [issues here](https://github.com/terraform-azurerm-avm-ptn-virtualwan/issues).
```

Comment:
typo?

Reply:
Interesting, that was a grept update.

Reply:
My mistake, I was setting the env vars wrong for grept. They should be:

`$env:GITHUB_REPOSITORY="Azure/terraform-azurerm-avm-ptn-virtualwan"`
`$env:GITHUB_REPOSITORY_OWNER="Azure"`

I assumed that GITHUB_REPOSITORY was just the repository name...